### DISCUSSION TITLE
Thank you for MS4

### DISCUSSION THREAD

**Discussion by DaddyLudwig** [musescore/MuseScore#17084] [General]
I love MS4, have since the beta. And it's getting better and better all the time. Thank you for the implementation of Muse Sounds playback. Thank you for developing MuseScore and continuing to do so. And most of all, thank you for making saving files with foreign language characters like German umlauts easier in MS4. I no longer have to deal with ae, oe, and ue diphthongs taking place of my umlauts and having to copy and paste the umlauts from somewhere else to put in the file name, it just saves with the umlauts(Ex. Die Schoene Muellerin in MS3 vs Die Schöne Müllerin in MS4 as the default save name for the file) 

Thank you very much for that. As someone who transcribes a lot of classical music and runs into a lot of German titles, this means a big deal to me that MS4 saves the umlauts by default if they are in the score title instead of replacing them with diphthongs the way MS3 does.

**Comment by shoogle** [comment]
You're welcome! Thanks for the great feedback. I hope we can continue to make it better for you!
